frontend/appflowy_flutter/lib/workspace/presentation/settings/pages/settings_shortcuts_view.dart

```diff
@@ -384,40 +398,73 @@ class _ShortcutSettingTileState extends State<ShortcutSettingTile> {
     );
   }
 
-  Widget _renderKeybindings(bool isHovering) => Row(
-        children: [
-          if (widget.command.keybindings.isNotEmpty) ...[
-            ..._toParts(widget.command.keybindings.first).map(
-              (key) => KeyBadge(keyLabel: key),
-            ),
-          ] else ...[
-            const SizedBox(height: 24),
-          ],
-          const Spacer(),
-          if (isHovering)
-            GestureDetector(
-              onTap: () {
-                if (widget.canStartEditing()) {
-                  setState(() {
-                    widget.onStartEditing();
-                    isEditing = true;
-                  });
-                }
-              },
-              child: MouseRegion(
-                cursor: SystemMouseCursors.click,
-                child: FlowyTooltip(
-                  message: LocaleKeys.settings_shortcutsPage_editTooltip.tr(),
-                  child: const FlowySvg(
-                    FlowySvgs.edit_s,
-                    size: Size.square(16),
+  Widget _renderKeybindings(bool isHovering) {
+    final canReset = canResetCommand(widget.command);
+
+    return Row(
+      children: [
+        if (widget.command.keybindings.isNotEmpty) ...[
+          ..._toParts(widget.command.keybindings.first).map(
+            (key) => KeyBadge(keyLabel: key),
+          ),
+        ] else ...[
+          const SizedBox(height: 24),
+        ],
+        const Spacer(),
+        if (isHovering)
+          Row(
+            children: [
+              GestureDetector(
```

Comment:
There's a new button named `PrimaryRoundedButton` I think you can replace this code with it with a better UI style.

Or just use `FlowyButton` which proivides hover effect and click action if you don't need a primary background color.

Reply:
So I don't want a background color thus I can not use the `PrimaryRoundedButton` and `FlowyButton` does not seem to provide a way to show a tooltip upon hovering on it. Thus I am sticking with the existing implementation. Let me know if you have something better than it.